src/test/java/edu/hm/hafner/analysis/FingerprintGeneratorTest.java

```diff
@@ -184,7 +184,7 @@ private void assertThatIssueHasDefaultFingerprint(final Report report) {
     private FullTextFingerprint createFullTextFingerprint(final String secondFile) {
         var fileSystem = stubFileSystem("fingerprint-one.txt", secondFile);
 
-        return new FullTextFingerprint(fileSystem);
+        return new FullTextFingerprint(fileSystem, 3);
```

Comment:
Can you add an additional test like `FullTextFingerprintTest#shouldExtractCorrectLines` that shows that a different value like 1 produces the correct result?